main.go

```diff
@@ -199,22 +202,20 @@ func getCacheExcludedObjectsTypes() []client.Object {
 	}
 }
 
-func getCacheFuncOptions() (*cache.Options, error) {
+func getCacheOptions() cache.Options {
 	componentPipelineRunRequirement, err := labels.NewRequirement(controllers.ComponentNameLabelName, selection.Exists, []string{})
 	if err != nil {
-		return nil, err
+		panic(err)
```

Comment:
Why switch from returning an error to `panic`?

Reply:
Because the operation above is safe. It can error if we configure it wrongly, but that should be fixed in the same commit and shouldn't bother the caller.

Reply:
Ack :+1: 

Could you add a comment about that or explain it in the commit message?

Reply:
ok